cmd/state-installer/cmd.go

```diff
@@ -244,16 +240,14 @@ func execute(out output.Outputer, cfg *config.Instance, an analytics.Dispatcher,
 		}
 	}
 
-	installerPath := filepath.Dir(osutils.Executable())
-	if params.sourcePath == "" {
-		params.sourcePath = installerPath
-	}
+	// We expect the installer payload to be in the same directory as the installer itself
+	params.payloadPath = filepath.Dir(osutils.Executable())
```

Comment:
`params` is for params the user can pass via flags. Since the user isn't passing this we shouldn't be using it. Just use a regular var that you pass around.